nautobot/docs/development/homepage.md

```diff
@@ -0,0 +1,72 @@
+# Populating the Homepage
+
+Both core applications and plugins can contribute items to the homepage by defining `layout` inside of their app's `homepage.py`. Using key and weight system, a developer can integrate amongst existing homepage objects or can create entirely new objects as desired.
+
+## Modifying Existing Objects
+
+By defining an object with the same identifier, a developer can modify existing objects. The object below shows modifying an existing object to have a custom template.
+
+!!! tip
+    Weights for already existing items can be found in the nautobot source code (in `homepage.py`) or with a web session open to your nautobot instance, you can inspect an element of the navbar using the developer tools.
```

Comment:
Actually, it looks like the `home.html` rendering is *not* currently including the weights in the output - need to steal some code from `nav_menu.html` to add `data-*-weight` attributes as appropriate.